### PR TITLE
Add resolution level to data access

### LINKED ISSUE #7: Add resolution level support
Add this feature from the base repo 
https://github.com/bioio-devs/bioio-base/pull/10

### PR DESCRIPTION
**Pull request by SeanLeRoy**
### Link to Relevant Issue

This pull request resolves #7

### Description of Changes

This adds support for getting and setting resolution levels

#### Testing
Tested successfully locally with changes from the required partner PR [here](https://github.com/bioio-devs/bioio-base/pull/10)

### CODE REVIEW

**Comment by SeanLeRoy** [comment]
CI won't pass until this other PR in bioio-base is merged and released https://github.com/bioio-devs/bioio-base/pull/10

File: bioio_ome_zarr/reader.py [hunk 1]
```diff
@@ -99,14 +98,27 @@ def scenes(self) -> Tuple[str, ...]:
...
+
     def _read_delayed(self) -> xr.DataArray:
         return self._xarr_format(delayed=True)
 
     def _read_immediate(self) -> xr.DataArray:
         return self._xarr_format(delayed=False)
 
     def _xarr_format(self, delayed: bool) -> xr.DataArray:
-        image_data = self._zarr.load(str(self.current_scene_index))
+        data_path = self._zarr.root_attrs["multiscales"][self.current_scene_index]["datasets"][self.current_resolution_level]["path"]
```
Comment (toloudis) [inline bioio_ome_zarr/reader.py:120]: Nit: maybe check that the level is valid before indexing into datasets here?

**Review by toloudis — APPROVED** [review]
LGTM

**Review by evamaxfield — APPROVED** [review]

File: bioio_ome_zarr/reader.py (hunk 1 above)
Comment (SeanLeRoy) [reply bioio_ome_zarr/reader.py:120]: Hmm do you think that is necessary with `set_current_resolution_level()` checking to see if it is valid at set time?

**Pull request merged**